src/core/reference.js

```diff
@@ -5,7 +5,15 @@
  */
 
 /**
+ * Creates and names a new variable. A variable is a container for a value.
+ * Variables that are declared with <a href="#/p5/let">let</a> will have block-scope.
+ * This means that the variable only exists within the <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/block">
+ * block</a> that it is created within.
+ *
+ *
+ * From <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let">the MDN entry</a>:
  * Declares a block scope local variable, optionally initializing it to a value.
+ *
  * @property let
```

Comment:
not totally sure `@property` is the right tag, but I couldn't easily see [which yuidoc tag](https://yui.github.io/yuidoc/syntax/index.html) would be more fitting. so if this works best with our reference building, let's use it!

Reply:
Yeah I am unsure either. I think none of them are perfect but I will experiment with other tags in case something feels a little closer to correct.

Reply:
after thinking about it some more, let's stick with the tags you used unless there's another that makes considerably more sense. I have a feeling using `property` will just play a little nicer with the yuidocs / reference build system we've built

Reply:
Sounds good!